src/main/java/aquarium/shop/basket/Basket.java

```diff
@@ -0,0 +1,61 @@
+package aquarium.shop.basket;
+
+import aquarium.shop.product.Product;
+import java.util.HashMap;
+
+
+public class Basket {
```

Comment:
One thing is missing: some kind of **showBasket** method should be implemented.
It should return all products in basket (your hash map) in some nice way.
Apart from products and their quantities we need also whole price (this should be calculated and also stored).

Maybe some new object should be created and returned to represent all requirements?